spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesInformerDiscoveryClient.java

```diff
@@ -238,22 +237,7 @@ public List<String> getServices() {
 
 	@PostConstruct
 	public void afterPropertiesSet() {
-		sharedInformerFactories.forEach(SharedInformerFactory::startAllRegisteredInformers);
```

Comment:
this PR does not do anything "meaningful", it just moves some code to the utils, so that reading the client becomes easier. It also replaces the deprecated `Wait` and moves some logging to the `else` branch.